App.js

```diff
@@ -18,11 +24,20 @@ export default class App extends React.Component {
       "Nunito-Bold": require("./assets/fonts/Nunito-Bold.ttf"),
       "Nunito-Black": require("./assets/fonts/Nunito-Black.ttf")
     });
-    this.setState({ fontLoaded: true });
+    setTimeout(() => {
+      this.setState({ fontLoaded: true });
+    }, 400);
```

Comment:
This looks hacky. As a user, I wouldn't wait to wait an extra half second anytime I launched the app just so fonts could load.

That being said, I can appreciate that you likely would rather about other things though for the purpose of this project, and this seems like a fine corner to cut so you can show us more important things.

Reply:
Definitely hacky!

Without this the fonts load in sub 100ms which causes the animation to appear as "just a flicker". 400ms allows for one full run of the `.gif` loading animation. I could get rid of this timeout to improve load times but I actually think the flicker makes for a worse user experience.

The ideal scenario would just be to display our splash screen until initial application render is 100% ready. I will look into this as a future enhancement

Reply:
That makes a lot of sense. It can be a bad experience if a splash screen just flickers. I didn't realize that was the reason for this time out but it makes sense now!